### PR TITLE
data: PriyanshShrivasatava

### PR DESCRIPTION
**Pull request by PriyanshShrivastava**
docs: adding my (Priyansh Shrivastava) data in LinkFree repo.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4931"></a>

### CODE REVIEW

File: data/PriyanshShrivastava.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Priyansh Shrivastava",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Full Stack Developer | DevOps | Javascript | Node.js | TailwindCSS ",
+	"avatar": "https://github.com/PriyanshShrivastava.png",
```
Comment (sansarmaske) [inline data/PriyanshShrivastava.json:6]:
```suggestion
```
`avatar` is no longer needed. Picture will be fetched automatically from your GitHub account

File: data/PriyanshShrivastava.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		"icon": "SiHashnode"
+	}, {
+		"name": "Some animated stuff here",
+		"url": "https://codepen.io/PriyanshShrivastava",
+		"icon": "SiCodepen"
+	}, {
+		"name": "Some basic Blockchain content ",
+		"url": "https://youtube.com/@inyour20s46",
+		"icon": "FaYoutube"
+	}],
```
Comment (sansarmaske) [inline data/PriyanshShrivastava.json:31]:
```suggestion
"links": [
    {
      "name": "like humour? Follow me on Twitter",
      "url": "https://twitter.com/shadow_priyansh",
      "icon": "FaTwitter"
    },
    {
      "name": "Let's collaborate on some amazing project",
      "url": "https://github.com/PriyanshShrivastava",
      "icon": "FaGithub"
    },
    {
      "name": "Follow me on LinkedIn for some achievement bragging!",
      "url": "https://www.linkedin.com/in/priyansh-shrivastava-184355145/",
      "icon": "FaLinkedin"
    },
    {
      "name": "Read my blogs on Hashnode",
      "url": "https://priyanshshrivastava.hashnode.dev/",
      "icon": "SiHashnode"
    },
    {
      "name": "Some animated stuff here",
      "url": "https://codepen.io/PriyanshShrivastava",
      "icon": "SiCodepen"
    },
    {
      "name": "Some basic Blockchain content ",
      "url": "https://youtube.com/@inyour20s46",
      "icon": "FaYoutube"
    }
  ],	
```

**Review by sansarmaske — CHANGES_REQUESTED** [review]

File: data/PriyanshShrivastava.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Priyansh Shrivastava",
+	"displayStatsPublic": true,
```
Comment (imbeshat) [inline data/PriyanshShrivastava.json:3]:
```suggestion
```

Comment (imbeshat) [inline data/PriyanshShrivastava.json:3]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please, remove this also as it is not required.

**Review by Hrishi5111998 — APPROVED** [review]

**Review by lewisushindi — APPROVED** [review]

**Review by stevenovak123 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**